### PR TITLE
changes to speed up matrix extraction massively.

### PR DESCRIPTION
**Pull request by alan-stokes**
this moves te setting of the system routes and the timeouts from a machine vertex level to the projection level. MASSIVELY speeds up matrix extraction

JENKINS: https://github.com/SpiNNakerManchester/sPyNNaker8/pull/419

### CODE REVIEW

File: spynnaker/pyNN/models/neuron/synaptic_manager.py [hunk 1]
```diff
@@ -1322,7 +1313,7 @@ def get_connections_from_machine(
             transceiver, placement, master_pop_table, indirect_synapses,
             direct_synapses, key, pre_vertex_slice.n_atoms, index,
             using_extra_monitor_cores, placements, monitor_api,
-            extra_monitor, monitor_cores, fixed_routes=fixed_routes)
```
Comment (andrewgait) [inline spynnaker/pyNN/models/neuron/synaptic_manager.py:1325]: extra_monitor should still be in the argument list here

Comment (alan-stokes) [reply spynnaker/pyNN/models/neuron/synaptic_manager.py:1325]: confused. everywhere else it was removing 2 elements. why would this one only need to remove 1? also wheres the handle_time_out_config? ug. 

but the tests pass.......... very very confused

Comment (andrewgait) [reply spynnaker/pyNN/models/neuron/synaptic_manager.py:1325]: The integration tests don't pass - check the equivalent spynnaker8 branch...

Comment (alan-stokes) [reply spynnaker/pyNN/models/neuron/synaptic_manager.py:1325]: doh. tis good to renember theres a 8 at the end of that repo. ;-)

Comment (alan-stokes) [reply spynnaker/pyNN/models/neuron/synaptic_manager.py:1325]: good catch by the way on the bug

**Pull request merged**